.gitignore

```diff
@@ -184,3 +184,4 @@ $RECYCLE.BIN/
 
 # JetBrains Rider
 .idea/
+.cr/Settings.xml
```

Comment:
why this change?

Reply:
Should have been in my local `.gitignore`